src/vstest.console/Resources/Resources.resx

```diff
@@ -728,6 +728,10 @@
   <data name="ProgressIndicatorString" xml:space="preserve">
     <value>Test run in progress</value>
   </data>
+  <data name="InvalidArgument" xml:space="preserve">
+    <value>The argument {0} provided is incorrect or invalid. Kindly use /help option to check the valid args
+Know more at : https://docs.microsoft.com/en-us/visualstudio/test/vstest-console-options</value>
```

Comment:
please indent this, see above(691) for reference.

Reply:
indented the lines